### DISCUSSION TITLE
Please give me this tutorial link (nested dropdown)

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#661] [Q&A] [answered]
hi
please give me link of tutorial that describe how to implement two drop down field in component when select some value from dropdown1 then change dropdown2 field values . for example select country then show it,s cites. (ajaxify) 
i already see that but i can not find that again.
thank you @Llewellynvdm

**Comment by Llewellynvdm** [comment, marked as answer]
Please search the [wiki](https://github.com/vdm-io/Joomla-Component-Builder/search?q=country+region&type=wikis) as all the videos tutorials are converted to text and are in the wiki for that reason.

**Reply by Llewellynvdm** [reply]
https://github.com/vdm-io/Joomla-Component-Builder/search?q=country+region&type=wikis
